wiki/Beatmap_Editor/Distance_Snap/en.md

```diff
@@ -1,3 +1,12 @@
+---
+tags:
+  - ds
+---
+
+```yaml
+needs_cleanup: true
+```
+
 # Distance snap
```

Comment:
This needs a definition/description of sorts under this. I'm pretty sure the first paragraph under "Using distance snap" works fine for this.